usr.sbin/bsdinstall/scripts/jail

```diff
@@ -108,20 +116,46 @@ fi
 
 bsdinstall checksum || error "Distribution checksum failed"
 bsdinstall distextract || error "Distribution extract failed"
-bsdinstall rootpass || error "Could not set root password"
+if [ -z "$SCRIPT" ]
+then
+    bsdinstall rootpass || error "Could not set root password"
+fi
+
+if [ -n "$SCRIPT"]
+then
+    # Configure bootloader if needed
+    bsdinstall bootconfig
```

Comment:
I don't think it makes sense to install the boot config in a jail

Reply:
It comes from https://github.com/freebsd/freebsd-src/blob/main/usr.sbin/bsdinstall/scripts/script#L151

Reply:
done